packages/python-sdk/tests/async/sandbox_async/files/test_write.py

```diff
@@ -1,10 +1,16 @@
 from e2b import AsyncSandbox
-
+from e2b.sandbox_async.filesystem.filesystem import EntryInfo
 
 async def test_write_file(async_sandbox: AsyncSandbox):
     filename = "test_write.txt"
     content = "This is a test file."
 
+    # Attempt to write without path
+    try:
+        await async_sandbox.files.write(None, content)
+    except Exception as e:
+        assert "object is not iterable" in str(e)
```

Comment:
Can we explicitly mention in the error message that a user forgot to specify a path?